File: Python/4 1차원 배열/5597.py
# https://www.acmicpc.net/problem/5597

import sys
input = sys.stdin.readline

students = [0] * 30
for _ in range(28):
    students[int(input())-1] = 1
for i in range(30):
    if students[i] == 0:
        print(i+1)

File: C++/4 1차원 배열/5597.cpp
// https://www.acmicpc.net/problem/5597

#include <stdio.h>

int main()
{
    int arr[30];
    for (int i = 0; i < 28; i++)
    {
        int n;
        scanf("%d", &n);
        arr[n - 1] = 1;
    }

    int cnt=0;
    for (int i = 0; i < 30; i++)
    {
        if (arr[i] != 1)
        {   if (cnt == 0)
                printf("%d\n", i + 1);
            else
                printf("%d", i + 1);
            cnt++;
        }   
    }
}

